src/domain/racing/racing.model.js

```diff
@@ -0,0 +1,73 @@
+import {
+  INITIAL_CAR_LIST,
+  INITIAL_RACING_HISTORY,
+  INITIAL_ROUND,
+  INITIAL_WINNER_LIST,
+  RACING_ROUND,
+} from "./racing.constant.js";
+
+export class Racing {
+  #round;
+  #carList;
+  #winnerList;
+  #movementRule;
+  #racingHistory;
+
+  constructor({ carList = INITIAL_CAR_LIST, movementRule }) {
+    this.#carList = carList;
+    this.#movementRule = movementRule;
+    this.#round = INITIAL_ROUND;
+    this.#winnerList = INITIAL_WINNER_LIST;
+    this.#racingHistory = INITIAL_RACING_HISTORY;
+  }
+
+  start() {
+    this.#race();
+    this.#determineWinnerList();
+  }
+
+  #race() {
+    while (this.#round < RACING_ROUND) {
+      this.#executeRound();
+      this.#updateRacingHistory();
+    }
+  }
+
+  #executeRound() {
+    this.#carList.forEach((car) => {
+      car.move(this.#movementRule());
+    });
```

Comment:
의존성이 잘 분리되었네요 👍 

추가로, 앞으로 이동하는지에 대한 조건을 move 밖으로 빼면 "move"의 역할이 더 명료해질 것 같다는 생각이 들었어요 ㅎㅎ
```suggestion
    this.#carList.forEach((car) => {
      if (this.#movementRule()) {
        car.move();
      }
    });
```